### PR TITLE
Gazebo7 Support

### PR DESCRIPTION
**Pull request by costashatz**
This pull-request combines #5 and #4 to provide support for newer versions of Gazebo without breaking compatibility with previous versions...

I tested in Gazebo 2 and 3 and it works. Thanks to @dasg and @bmagyar, I am sure it works in newer versions of Gazebo (>5)..

### CODE REVIEW

**Comment by costashatz** [comment]
I also made some formatting changes...

**Comment by vrabaud** [comment]
Thx! Careful with the reformatting. Files that don't change except for spaces should not be part of this pull request.

**Comment by costashatz** [comment]
> Files that don't change except for spaces should not be part of this pull request.

That's true, but it is a very small library. I can revert the formatting if you like and put it in a different PR.

**Comment by vrabaud** [comment]
Well, mostly, it makes it hard to follow what happened. But wait !
https://github.com/blog/967-github-secrets
Enjoy:
https://github.com/roboticsgroup/roboticsgroup_gazebo_plugins/pull/7/files?w=1

**Comment by costashatz** [comment]
Should be better now..! Should I merge?

**Comment by vrabaud** [comment]
LGTM !

**Comment by costashatz** [comment]
**Merging...**

**Comment by bmagyar** [comment]
Great guys, thanks!
Any chance we can get a release too?
